dags/author/author_create/author_create_init.py

```diff
@@ -46,7 +46,7 @@ def set_workflow_status_to_running(**context):
 
     @task()
     def create_author_create_user_ticket(**context: dict) -> None:
-        endpoint = "/tickets/create-with-template"
+        endpoint = "/api/tickets/create"
```

Comment:
why we change the api endpoint?

Reply:
due to a change in inspirehep that pascal has done that was not yet updatted

https://github.com/inspirehep/inspirehep/pull/3097/files#diff-147704379813427143b24ca6b9e81b711aae49900a7fe939dbc6d131c9961708L58

Equivalent implementation in next:
https://github.com/inspirehep/inspire-next/pull/4333/files